pkg/controller/handler.go

```diff
@@ -93,12 +93,19 @@ func (h *handler) createServiceInstance(in *servicecatalog.Instance) error {
 
 	client := h.newClientFunc(broker.Name, broker.Spec.URL, authUsername, authPassword)
 
-	// TODO: uncomment parameters line once parameters types are refactored.
-	// Make the request to instantiate.
+	parameters := make(map[string]interface{})
+	if len(in.Spec.Parameters.Raw) > 0 {
+		err = yaml.Unmarshal([]byte(in.Spec.Parameters.Raw), &parameters)
```

Comment:
The unmarshal here and below are to change from bytes (of text) to a struct. Why a yaml unmarshal vs the json unmarshal?